src/session/session_compact.c

```diff
@@ -260,17 +262,12 @@ __wt_session_compact_check_interrupted(WT_SESSION_IMPL *session)
 
 /*
  * __compact_checkpoint --
- *     This function does wait and force checkpoint.
+ *     This function waits and triggers a checkpoint.
  */
 static int
 __compact_checkpoint(WT_SESSION_IMPL *session)
 {
-    /*
-     * Force compaction checkpoints: we don't want to skip it because the work we need to have done
-     * is done in the underlying block manager.
-     */
-    const char *checkpoint_cfg[] = {
-      WT_CONFIG_BASE(session, WT_SESSION_checkpoint), "force=1", NULL};
+    const char *checkpoint_cfg[] = {WT_CONFIG_BASE(session, WT_SESSION_checkpoint), NULL, NULL};
```

Comment:
No need of this variable as we are not passing any config options, passing NULL is same as passing default config options.

Reply:
I don't think this is true. Since we are calling the internal `__wt_txn_checkpoint`, we need to give at least the base config.

Reply:
Thanks for the correction. I agree that it is required.